configure.py

```diff
@@ -1766,6 +1774,29 @@ def configure_inspector(o):
                        options.without_ssl)
   o['variables']['v8_enable_inspector'] = 0 if disable_inspector else 1
 
+def configure_section_file(o):
+  try:
+    proc = subprocess.Popen(['ld.gold'] + ['-v'], stdin = subprocess.PIPE,
+                            stdout = subprocess.PIPE, stderr = subprocess.PIPE)
+  except OSError:
+    return 0
+
+  match = re.match(r"^GNU gold.*([0-9]+)\.([0-9]+)$",
+                   proc.communicate()[0].decode("utf-8"))
+
+  if match:
+    gold_major_version = match.group(1)
+    gold_minor_version = match.group(2)
+    if int(gold_major_version) == 1 and int(gold_minor_version) <= 1:
+      error('''GNU gold version must be greater than 1.2 in order to use section
+            reordering''')
+
+  if options.node_section_ordering_file != "":
+    o['variables']['node_section_ordering_file'] = os.path.realpath(
+      str(options.node_section_ordering_file))
+  else:
+    # An empty string here will cause gyp to fail.
```

Comment:
That's because gyp handles variables ending in `_file` (and `_dir` and `_path`) specially. Rename it and it should work without this workaround.